test/interop/python/chapelBytes/PREDIFF

```diff
@@ -13,7 +14,9 @@ sed '/ndarraytypes.h:[0-9]*/d' | \
 sed '/Using deprecated NumPy API/d' | \
 sed '/^ *\^/d' | \
 sed '/^1 warning generated./d' | \
+sed '/^[2-9] warnings generated./d' | \
```

Comment:
Can these files all be symbolic links to test/interop/python/PREDIFF to make maintenance easier?